docs/writing-rules/glossary.md

```diff
@@ -0,0 +1,124 @@
+---
+slug: glossary
+title: SAST and rule-writing glossary
+hide_title: true
+description: Definitions of static analysis and Semgrep rule-writing terms.
+tags:
+  - Rule writing
+---
+
+# Static analysis and rule-writing glossary
+
+The definitions provided here are specific to each term's meaning and use in Semgrep.
```

Comment:
Which "same thing" are you referring to here?

Reply:
Specifically the line "The definitions provided here are specific to each term's meaning and use". I'd be confused because if both are specific to Semgrep, why is the Rule-writing glossary different?